applications/main/subghz/subghz_cli.c

```diff
@@ -1064,16 +1064,26 @@ static void subghz_cli_command_chat(Cli* cli, FuriString* args) {
                     memset(message, 0x00, message_max_len);
                     size_t len = subghz_chat_worker_read(subghz_chat, message, message_max_len);
                     for(size_t i = 0; i < len; i++) {
-                        furi_string_push_back(output, message[i]);
                         if(message[i] == '\n') {
-                            printf("\r");
-                            for(uint8_t i = 0; i < 80; i++) {
+                            // Check it was \r before
+                            if((i > 0 && message[i - 1] != '\r') || i == 0) {
```

Comment:
this time you overdid yourself: now layout is a total mess. Check with `./fbt cli`

Reply:
I mean what exactly you were trying to fix there?